dubbo-config/dubbo-config-spring/src/main/java/org/apache/dubbo/config/spring/util/DubboAnnotationUtils.java

```diff
@@ -154,7 +150,7 @@ public static String resolveInterfaceName(Reference reference, Class<?> defaultI
      */
     public static Map<String, String> convertParameters(String[] parameters) {
         if (ArrayUtils.isEmpty(parameters)) {
-            return Collections.emptyMap();
+            return new HashMap<>();
```

Comment:
Null return in `org.apache.dubbo.config.AbstractMethodConfig#getParameters` is also need to be considered